# HG changeset patch
# User John D
# Date 1486782976 18000
#      Fri Feb 10 22:16:16 2017 -0500
# Node ID 8db35b1a4f63b364f3e5756cb2d28e31be881da2
# Parent  dfee8aa390c229f03f4cc6fc391410dda6798381
build: Clear ac_cv_prog_ac_ct_XXXX qt variables if correct qt was not found (Bug #50214)

* m4/acinclude.m4: clear ac_cv_prog_ac_ct_MOC_QTVER, ac_cv_prog_ac_ct_UIC_QTVER,
    ac_cv_prog_ac_ct_RCC_QTVER and ac_cv_prog_ac_ct_LRELEASE_QTVER if all tools not found

diff --git a/m4/acinclude.m4 b/m4/acinclude.m4
--- a/m4/acinclude.m4
+++ b/m4/acinclude.m4
@@ -1520,19 +1520,23 @@ AC_DEFUN([OCTAVE_CHECK_QT_VERSION], [AC_
       UIC_QTVER=
       RCC_QTVER=
       LRELEASE_QTVER=
       MOC_FLAGS=
       UIC_FLAGS=
       RCC_FLAGS=
       LRELEASE_FLAGS=
       $as_unset ac_cv_prog_MOC_QTVER
+      $as_unset ac_cv_prog_ac_ct_MOC_QTVER
       $as_unset ac_cv_prog_UIC_QTVER
+      $as_unset ac_cv_prog_ac_ct_UIC_QTVER
       $as_unset ac_cv_prog_RCC_QTVER
+      $as_unset ac_cv_prog_ac_ct_RCC_QTVER
       $as_unset ac_cv_prog_LRELEASE_QTVER
+      $as_unset ac_cv_prog_ac_ct_LRELEASE_QTVER
     fi
   fi
 
   if test $build_qt_gui = yes; then
     AC_CHECK_FUNCS([setlocale], [],
       [build_qt_gui=no
        warn_qt_setlocale="setlocale not found; disabling Qt GUI"])
   fi
